Engines/Wine/Engine/Versions/script.js

```diff
@@ -122,15 +123,24 @@ module.getAvailableVersions = function (wizard) {
 
 
 module.getLatestStableVersion = function (wizard, architecture) {
-    return getLatestVersion(wizard, "upstream-linux-" + architecture, /^\d+\.0(\.\d+)?$/);
+    const os = operatingSystemFetcher.fetchCurrentOperationSystem().getWinePackage()
+    const arch = operatingSystemFetcher.fetchCurrentOperationSystem().getWinePackage() === "darwin" ? "x86on64" : "x86";
+    const distribution = operatingSystemFetcher.fetchCurrentOperationSystem().getWinePackage() === "darwin" ?  "cx" : "upstream";
```

Comment:
Pass the `distribution` as additional parameter (like `architecture`). See https://github.com/PhoenicisOrg/scripts/blob/7eb38913dff8fcacbd5104ec6b3e247a0c5b7897/Engines/Wine/QuickScript/Quick%20Script/script.js#L157.